CHANGELOG.md

```diff
@@ -1,8 +1,6 @@
 # Change log
 
-All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
```

Comment:
This doesn't really need changed since it will get taken from the HISTORY.md.
Also, should the link info and release date be added below it? Just to maintain the info?

Reply:
When running the changelog generator this is what it done I haven't made any manual changes.
As it is automated if I was to add back in the information it will be removed again. 
It is just how it works, in order for it to pick it up I needs to be v11.0.3 with nothing else.

Reply:
Didn't think youd already ran it, looks good to me then